### ISSUE TITLE
[docker.md] This page is a bit out of date

### ISSUE THREAD

**Issue by rkoshak** [openhab/openhab-docs#1953]

**Comment by Confectrician** [comment]
So, clean start in the docs repo now.

If i am right the docker repo can tehoretically be independent from the openhab release.
Mostly this won't make sense, but theoretically it is possible.
So we could simply handle it's `Readme.md` like any other external tool like google assistant, openhabian or alexa.

We have to adapt the script a bit for this repo probably, but it should be possible.
We can also create a ruby script or function to strip of the contents that are special for dockerhub, but are not needed on the docs website.

**Comment by rkoshak** [comment]
> If i am right the docker repo can tehoretically be independent from the openhab release.

Yes, that's true. We keep running into that more and more, don't we.

In truth, the logo and stuff at the top for Dockerhub doesn't bother me much. Definitely not as much as making sure we have stuff in the main docs like the docker-compose example, how to pass through a device, and stuff like that. So if it makes sense to pull it in as in and leave for a little later stripping off the top stuff I for one would think that's just fine.
